pkgs/development/compilers/blueprint/default.nix

```diff
@@ -0,0 +1,39 @@
+{ python3
+, stdenv
+, fetchFromGitLab
+, gobject-introspection
+, lib
+, meson
+, ninja
+}:
+stdenv.mkDerivation rec {
+  pname = "blueprint-compiler";
+  version = "0.pre+date=2022-05-17";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "jwestman";
+    repo = pname;
+    rev = "06278a64248cec92bb95a958eadfba453943c061";
+    sha256 = "sha256-ukvqWvYl4NohKEm6W50eRZ9fmKD4vhH0yzgrCf5HVts=";
+  };
+
+  # Requires pythonfuzz, which I've found difficult to package
+  doCheck = false;
```

Comment:
Maybe we should at least have a `pythonImportsCheck` here?

While it will not work by default on `stdenv.mkDerivation`, it should work if you add `python3.pkgs.pythonImportsCheckHook` to `nativeBuildInputs`.

Reply:
Huh... I think because this is not a `pythonModule` them.

Ok, fine by me.

Reply:
There is a module `blueprintcompiler` in the source code.

It's just that I can't get it imported with `pythonImportsCheck`

Reply:
I am not sure this is necessary though. If the only use is the application itself the way it currently is fine.